src/sentry/incidents/action_handlers.py

```diff
@@ -120,12 +139,20 @@ def generate_email_context(self, status):
                 )
             ),
             "incident_name": self.incident.title,
+            # TODO(alerts): Add environment
+            "environment": "All",
+            "time_window": format_duration(alert_rule.time_window),
```

Comment:
`time_window` is in minutes. Don't know why I did that, might refactor it at some point

Reply:
oops, updated https://github.com/getsentry/sentry/pull/16867 to be in minutes